### PR TITLE
Fuse on SpringBoot note

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/fuse/fuse-con.adoc [hunk 1]
```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
...
 ifdef::DM[]
 https://access.redhat.com/articles/3354301[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ====
+
+[NOTE]
+====
+You can also install {FUSE_LONG} on SpringBoot. {PRODUCT} provides no special integration for this scenario. However, your application that runs on {FUSE_LONG} on SpringBoot can use the `kie-server-client` library to communicate with {PRODUCT} services that run on a {KIE_SERVER}. The following limitations apply, because of library version conflicts between {FUSE_LONG} and {PRODUCT}:
+
+* You cannot use the JSON marshalling format, because of incompatible versions of Jackson.
```
Comment (tkobayas) [inline doc-content/enterprise-only/fuse/fuse-con.adoc:36]:
```suggestion
* You cannot use `MarshallingFormat.JSON`, because of incompatible versions of Jackson.
```
I think this is better because Fuse users can still use JSON for other purposes outside of `kie-server-client`. `MarshallingFormat.JSON` is clear so they can understand that simply they cannot use `MarshallingFormat.JSON` configuration of `kie-server-client`.

**Review by tkobayas — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/fuse/fuse-con.adoc [hunk 2]
```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
...
 https://access.redhat.com/articles/3405381[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ifdef::DM[]
 https://access.redhat.com/articles/3354301[{PRODUCT} 7 Supported Configurations].
 endif::[]
 ====
+
+[NOTE]
+====
+You can also install {FUSE_LONG} on SpringBoot. {PRODUCT} provides no special integration for this scenario. However, your application that runs on {FUSE_LONG} on SpringBoot can use the `kie-server-client` library to communicate with {PRODUCT} services that run on a {KIE_SERVER}. The following limitations apply, because of library version conflicts between {FUSE_LONG} and {PRODUCT}:
```
Comment (csherrar) [inline doc-content/enterprise-only/fuse/fuse-con.adoc:34]: [style] Correct "SpringBoot" to "Spring Boot", remove "also", and maybe rephrase to simplify. For example:

You can install {FUSE_LONG} on Spring Boot. You can use the `kie-server-client` library to enable communication between applications running on {FUSE_LONG} on {PRODUCT} with {PRODUCT} services running on {KIE_SERVER}. There are `kie-server-client` library version conflicts between {FUSE_LONG} and {PRODUCT} that cause the following limitations:

File: doc-content/enterprise-only/fuse/fuse-con.adoc [hunk 3]
```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
...
+
+* You cannot use `MarshallingFormat.JSON`, because of incompatible versions of Jackson.
```
Comment (csherrar) [inline doc-content/enterprise-only/fuse/fuse-con.adoc:36]: [style] Maybe avoid "You cannot use" and rephrase as per prerequisites statements?  Is it mismatched versions or incompatible version ranges? Is this a temporary version mismatch?
````
s/* You cannot use `MarshallingFormat.JSON`, because of incompatible versions of Jackson./* *Jackson version mismatch:* `MarshallingFormat.JSON` cannot be used.
````

File: doc-content/enterprise-only/fuse/fuse-con.adoc [hunk 4]
```diff
@@ -20,11 +20,21 @@ You can install {FUSE_LONG} on a separate instance of {EAP_LONG} and integrate i
...
+* You cannot use JMS messaging, because of incompatible versions of ActiveMQ.
```
Comment (csherrar) [inline doc-content/enterprise-only/fuse/fuse-con.adoc:37]: [style] Maybe avoid "You cannot use" and rephrase as per prerequisites statements?  Is it mismatched versions or incompatible version ranges? Is this a temporary version mismatch?
````
s/* You cannot use JMS messaging, because of incompatible versions of ActiveMQ./* *ActiveMQ version mismatch*: JMS messaging cannot be used.
````

**Review by csherrar — COMMENTED** [review]
A few minor suggestions, otherwise looks good, Misha!

File: doc-content/enterprise-only/fuse/fuse-con.adoc (hunk 2 above)
Comment (mramendi) [reply doc-content/enterprise-only/fuse/fuse-con.adoc:34]: I do believe that the "no special integration" part is important.

File: doc-content/enterprise-only/fuse/fuse-con.adoc (hunk 3 above)
Comment (mramendi) [reply doc-content/enterprise-only/fuse/fuse-con.adoc:36]: I do not think introducing the passive voice is better. (I did apply the other suggestions)

**Review by tkobayas — APPROVED** [review]
LGTM, thanks!
